Makefile

```diff
@@ -20,3 +23,14 @@ docker-build-eno-reconciler:
 		--file docker/$(ENO_RECONCILER_IMAGE_NAME)/Dockerfile \
 		--tag $(REGISTRY)/$(ENO_RECONCILER_IMAGE_NAME):$(ENO_RECONCILER_IMAGE_VERSION) .
 	docker push $(REGISTRY)/$(ENO_RECONCILER_IMAGE_NAME):$(ENO_RECONCILER_IMAGE_VERSION)
+
+# Run tests with increased timeout for all test packages
```

Comment:
This is not true, the default timeout for Go tests is 10m, but the default value for RECONCILIATION_TEST_TIMEOUT is 5m.